app/views/articles/index.html.haml

```diff
@@ -0,0 +1,11 @@
+- Category.all.each do |category|
+  = link_to category.name, root_path(category: category.id)
+- @articles.each do |article|
+  #header
+    %p= article.header
+  #subheader
```

Comment:
Again, why do we need those ids?

Reply:
We could use the category name instead. But using Ids we can change the names of the categories in the future (eg for language translation etc)

Reply:
My question is: What are these ids used for now? (if the answer is nothing, I would not have them)

Reply:
to navigate to a given category, which is defined by its id. but given we don't have specific categories set up yet, we could remove. but then our tests for categories would fail without this navigation (i think).